### ISSUE TITLE
Fix dep on Chef::PathHelper

### ISSUE THREAD

**Issue by danielsdeleo** [chef/chef-config#3]
As seen here and other places: https://github.com/chef/chef-config/blob/beeabc1f6cb1966e18cb4bd23ed66eca6f5aff14/lib/chef-config/config.rb#L90

**Comment by danielsdeleo** [comment]
@chef/client-core this is the trickiest of the remaining dependencies from config -> Chef. To me, looks like the cleanest way to do this is move PathHelper to here and then "alias" the class in core Chef, e.g..,

```ruby
class Chef
  module Util
    PathHelper = ChefConfig::PathHelper
  end
end
```

**Comment by tyler-ball** [comment]
:+1:

**Comment by danielsdeleo** [comment]
Fixed by #8
